R/get.R

```diff
@@ -0,0 +1,228 @@
+#' Get treaty articles by number or match
+#'
+#' The function splits treaty texts into lists that reflect a structure
+#' based on having a preamble and several articles.
+#' Once articles are split, users can access the "preambles",
+#' "accession" and "termination" clauses, or "annexes"
+#' within a collection of treaty texts.
+#' The selection of each of these articles relies on regex matching
+#' certain expressions unique to each of them.
+#' Alternatively, users can get all the articles that contain a certain
+#' 'desired' word within a collection of treaty texts.
+#' As well, users can select only certain types of treaties for which
+#' specific articles are to be returned (e.g. agreements).
```

Comment:
If this is user-facing, then perhaps it should go in manydata rather than manypkgs?

Reply:
The issue is that this function call many other code functions within the package as well as some NLP related package dependencies not available on CRAN, therefore, we cannot add these to manydata...